functions-and-scope.js

```diff
@@ -78,7 +117,18 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // Zorg ervoor dat jouw functie ook werkt als we een andere array willen checken, zoals bijvoorbeeld: [6, 4, 5] of [8, 9, 4, 6, 10].
 // Log het antwoord in de terminal.
 
+function highestGrade(grades) {
+    let highestGrade = 0
```

Comment:
Het is altijd beter om bij initiele waardes de eerste van de array te pakken i.p.v. zelf een waarde te verzinnen: `let hightestGrade = grades[0];`